src/coreclr/vm/stublink.cpp

```diff
@@ -1737,6 +1737,9 @@ bool StubLinker::EmitUnwindInfo(Stub* pStubRX, Stub* pStubRW, int globalsize, Lo
             (epilogUnwindCodeIndex << 22)|
             (codeWordsCount << 27);
     } // end else (!m_fProlog)
+
+#elif defined(TARGET_LOONGARCH64)
```

Comment:
This should not be needed. This code is compiled only for Windows

Reply:
Thanks.
I will revert it.